### DISCUSSION TITLE
missing some file like Migration.sol and 1_intial_demo.js

### DISCUSSION THREAD

**Discussion by Taaani** [ConsenSys-archive/truffle#5494] [Installation]
I install truffle globally and then use the command truffle init then some files are missing. one is Migration.sol and the other is 1_inital_js which gets by default if anyone Know about this problem kindly guide me

**Comment by apdan510** [comment]
Hey @Taaani 

Which version of Truffle are you using? Can you check by entering `truffle version` into your terminal? 

Thank you!

**Comment by Taaani** [comment]
Truffle v5.5.28 .

**Reply by apdan510** [reply]
Hey @Taaani we are currently in the process of updating the Truffle documentation to explain how migrations.sol is not needed in the latest version of Truffle, you will only need the regular deploy file. In the meantime here is a guide that shows that you do not need migrations.sol
https://trufflesuite.com/guides/nft-royalty/ 
Let me know if you have any other questions or concerns.
